### DISCUSSION TITLE
how to support sorting for columns with associated lookup values

### DISCUSSION THREAD

**Discussion by heswell** [finos/vuu#1313] [Ideas]
In the BASKET module included with the sample Vuu code, we have some tables with enum type numeric values with associated lookup tables. The lookup tables store user friendly labels for the raw enum values. We have an implementation of a lookup cell renderer in the ui code that will pull the lookup values from Vuu and render the appropriate labels in the ui. 

- how do we sort these columns ? The user would expect sorting to reflect the labels rendered in the ui not the raw  enum values.
- sorting is always serverside, so the clientside renderer wouldn't help here
- some apps will need multiple versions of labels for multiple language support
- joining lookup table to parent table might be a solution 
- a calculated column might be a solution, though not for virtualized tables (ie ignite backed tables)

Anyone have any thoughts on best way to deal with this.

**Comment by naleeha** [comment]
One of the suggestions mentioned was to allow user to provide Custom Sort Implementation for specific columns
